cmd/management.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	mgmt "github.com/wiretrustee/wiretrustee/management"
+	mgmtProto "github.com/wiretrustee/wiretrustee/management/proto"
+	"google.golang.org/grpc"
+	"net"
+)
+
+var (
+	mgmtPort    int
+	mgmtDataDir string
+
+	mgmtCmd = &cobra.Command{
+		Use:   "management",
+		Short: "start Wiretrustee Management Server",
+		Run: func(cmd *cobra.Command, args []string) {
+			flag.Parse()
+
+			lis, err := net.Listen("tcp", fmt.Sprintf(":%d", mgmtPort))
+			if err != nil {
+				log.Fatalf("failed to listen: %v", err)
+			}
+
+			if err != nil {
+				log.Fatalf("failed to listen: %v", err)
+			}
+
+			var opts []grpc.ServerOption
+			grpcServer := grpc.NewServer(opts...)
+			defer grpcServer.Stop()
+
+			server, err := mgmt.NewServer(mgmtDataDir)
+			if err != nil {
+				log.Fatalf("failed creating new server: %v", err)
+				panic(err)
+			}
+			mgmtProto.RegisterManagementServiceServer(grpcServer, server)
+			log.Printf("started server: localhost:%v", mgmtPort)
+			go func() {
+				if err = grpcServer.Serve(lis); err != nil {
+					log.Fatalf("failed to serve: %v", err)
+				}
+			}()
+
+			SetupCloseHandler()
+			<-stopCh
+			log.Println("Receive signal to stop running Management server")
+		},
+	}
+)
+
+func init() {
+	mgmtCmd.PersistentFlags().IntVar(&mgmtPort, "port", 33073, "Server port to listen on (e.g. 33073)")
+	mgmtCmd.PersistentFlags().StringVar(&mgmtDataDir, "datadir", "/data", "Server data directory location (e.g. /data")
```

Comment:
/data works well for docker. But account for the possibility of having the server running on non-docker env, we can set the default to /var/lib/wiretrustee. Or maybe follow the config check we got in the client to account for Windows systems, see https://github.com/wiretrustee/wiretrustee/blob/main/cmd/root.go#L34

Reply:
As the data dir is an initial implementation, we will skip moving to a different folder for now.